kubetest2/kubetest2-gce/deployer/up.go

```diff
@@ -135,3 +93,75 @@ func enableComputeAPI(project string) error {
 
 	return nil
 }
+
+// getProjectFromBoskos creates a boskos client, acquires a gcp project
+// and starts a heartbeat goroutine to keep the project reserved
+func (d *deployer) getProjectFromBoskos() error {
```

Comment:
maybe move all boskos methods to a different file, and try to remove deployer as a receiver as much as possible.

Reply:
Done.